Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1865,8 +1866,21 @@ def headerTransform(s): return stringEscapeMD(s, True, True)  # noqa
         sep = '---'
         mdResult += '|' + '|'.join([sep] * len(headers)) + '|\n'
         for entry in t:
-            vals = [stringEscapeMD((formatCell(entry.get(h, ''), False) if entry.get(h) is not None else ''),
+
+            entry_copy = entry.copy()
+
+            if date_fields:
+                for field in date_fields:
+
+                    try:
+                        entry_copy[field] = datetime.fromtimestamp(int(entry_copy[field])/1000).strftime('%Y-%m-%d %H:%M:%S')
+
```

Comment:
```suggestion
            entry_copy = entry.copy()
            if date_fields:
                for field in date_fields:
                    try:
                        entry_copy[field] = datetime.fromtimestamp(int(entry_copy[field])/1000).strftime('%Y-%m-%d %H:%M:%S')
```
try to put spaces only when you want to separate code blocks that handle different things